internal/server/v2/observation/contained_in_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package merger provides function to merge V2 API responses.
+package observation
+
+import (
+	"context"
+	"database/sql"
+	"net/http"
+	"reflect"
+	"testing"
+
+	pb "github.com/datacommonsorg/mixer/internal/proto"
+	pbv2 "github.com/datacommonsorg/mixer/internal/proto/v2"
+	"github.com/datacommonsorg/mixer/internal/server/resource"
+	"github.com/datacommonsorg/mixer/internal/store"
+)
+
+func TestFetchChildPlaces(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+
+	s := &store.Store{
+		SQLClient: &sql.DB{},
+	}
+	metadata := &resource.Metadata{}
+	httpClient := &http.Client{}
+
+	for _, tc := range []struct {
+		desc string
+		remoteMixer         string
+		ancestor            string
+		childType           string
+		storeResponse       map[string][]string
+		remoteMixerResponse *pbv2.NodeResponse
+		want                []string
+	}{{
+		desc: "store only",
+		remoteMixer: "",
+		ancestor: "a1",
+		childType: "CT1",
+		storeResponse: map[string][]string{"a1": {"c1", "c2"}},
```

Comment:
this test case does not represent current behavior since remoteResponse is always empty struct instead of nil now?

Reply:
Since `remoteMixerResponse` is a pointer, it's zero value should be `nil` so I think it represents current behavior?

Reply:
okay, now the nil pointer looks good.